problems/objects.js

```diff
@@ -72,7 +93,11 @@ const adamAndEveApples = () => {};
  *
  */
 
-const appleSum = () => {};
+const appleSum = (appleCountByName, number) => {
+  let sum = 0
+  sum = appleCountByName
```

Comment:
appleCountByName is an object. Make sure you think about things in terms of types of data.